### ISSUE TITLE
Single quotes not supported in sub expressions using a helper

### ISSUE THREAD

**Issue by devinus** [sunng87/handlebars-rust#232]
```thread 'main' panicked at 'Template error: invalid handlebars syntax.
    --> Template error in "filters/limited-loop":2:17
     |
   0 | {{#each components}}
   1 | 	{{#if (eq type 'filters')}}
     |-----------------^----------
   2 | 		{{> filters/limited.css productFilter=body style=style fallback=fallback}}
   3 | 		{{> filters/tracking.css productFilter=body }}
   4 | 	{{/if}}
   5 | 	{{#if components}}
     |
     = reason: invalid handlebars syntax.
', src/main.rs:96:9```

Changing this to double quotes seems to fix the issue.

**Comment by sunng87** [comment]
Thanks for reporting. Single quotes is not supported at the moment. Please use double quotes for now and I'll look into the parser this weekend.

**Comment by sunng87** [comment]
Fixed in 39bd58d
